### PR TITLE
Various tweaks

### LINKED ISSUE #68: Make output less verbose
I suggest outputting the total number of files that we processed, and only in verbose mode each one.

See https://travis-ci.org/mpc-hc/mpc-hc.org/builds/116440236 for example. Something like

> Processed 54 files

I'm willing to have a take at this tomorrow or so.

@alanshaw: your thoughts?

### PR DESCRIPTION
**Pull request by XhmikosR**
@alanshaw: please let me know if you want me to change something and I will rebase the branch.

On a side note, how do you feel about using chalk for coloring the output?

Fixes #68.

### CODE REVIEW

**Comment by XhmikosR** [comment]
Bump @alanshaw. Issue #68 is important in big projects. :/

**Comment by XhmikosR** [comment]
@alanshaw: please check this out and let me know if you want me to make any changes.

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ hello.html
 ```html
 <!DOCTYPE html>
 <h1>Hello World!</h1>
-@@include('/path/to/include/message.html', {"name": "Joe Bloggs"})
+@@include('/path/to/include/message.html', {'name': 'Joe Bloggs'})
```
Comment (alanshaw) [inline README.md]: This has to be valid JSON so please leave as is

Comment (XhmikosR) [inline README.md]: Done.

**Comment by alanshaw** [comment]
Thanks @XhmikosR :thumbsup:

**Comment by XhmikosR** [comment]
Np, @alanshaw. Not sure if you wanted colorized output or not; we could go with chalk later.

P>. Waiting for a new release :)

**Pull request merged**